app/Transformers/BeatmapSetTransformer.php

```diff
@@ -41,13 +43,35 @@ public function transform(BeatmapSet $beatmap = null)
             'artist' => $beatmap->artist,
             'play_count' => $beatmap->play_count,
             'favourite_count' => $beatmap->favourite_count,
+            'submitted' => $beatmap->submit_date->toIso8601String(),
```

Comment:
probably should be called `submitted_at` and `ranked_at` (or `_date`), especially the latter might be confused with a boolean if just `ranked`.